### ISSUE TITLE
Navigating to Settings > Form Behavior causes an error

### ISSUE THREAD

**Issue by mauricioarangoosorio** [solspace/craft-freeform#342]
**Describe the bug or issue you're experiencing**
I see a page error when navigating to Form Behavior

```
GuzzleHttp\Exception\ConnectException
cURL error 7: Failed to connect to mysite.local port 443 after 0 ms: Connection refused (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://mysite.local/freeform/plugin.js

```
**Steps to reproduce**

1. Install FreeForm 4 Beta 6
2. Go to Settings > Form Behavior

**Expected behavior**
See the page in question

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Solo 4.0.1
- Freeform Version: 4 beta 6
- Freeform Edition: Pro
- Fresh Install or Upgrade: Ugrade

**Additional context**
This is a Nitro setup.

**Comment by kjmartens** [comment]
Sorry for the trouble @mauricioarangoosorio. We recently added a check to see if Freeform's `plugin.js` file was accessible (and then alert you to use a different setting option if not). It seems we may have to tweak this a bit further. If you need to adjust any settings for that page in the meantime, you can do it via the Project Config file:
https://docs.solspace.com/craft/freeform/v4/setup/settings.html#form-behavior

Very sorry for the inconvenience, but we'll get it fixed soon.

**Comment by kjmartens** [comment]
This should be fixed in Freeform 4.0.0-beta.7. Would you be able to check and see if it's no longer an issue for you? 🙂
